ndcube/tests/test_ndcubesequence.py

```diff
@@ -140,7 +140,7 @@ def test_cube_like_shape(ndc, expected_shape):
                          ],
                          indirect=("ndc",))
 def test_cube_like_dimensions(ndc, expected_dimensions):
-    assert np.all(ndc.cube_like_dimensions == expected_dimensions)
+    assert all(assert_quantity_allclose(ndc_dim, exp_dim) for ndc_dim, exp_dim in zip(ndc.cube_like_dimensions, expected_dimensions))
```

Comment:
To avoid the deprecation warning making this test fail, I think you should be able to use Python's [`catch_warnings` context manager](https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings)